pylogix/eip.py

```diff
@@ -218,6 +222,12 @@ def Discover(self):
 
         returns Response class (.TagName, .Value, .Status)
         """
+        if is_micropython():
+            # limited implementation of lwip_socket_setsockopt()
+            # https://github.com/micropython/micropython/issues/2691
+            status = "Discover not available on micropython, due to limited socket module"
+            return Response(None, None, status)
```

Comment:
Small optimization - drop the unnecessary variable

```suggestion
            return Response(None, None, "Discover not available on micropython, due to limited socket module")
```

Reply:
Let's hold off on this, getting some suggestions on possibly making it work. But btw for the suggestion for me to commit the suggestion you'll need to delete the actual status variable. If I commit the suggestion the status variable will still be there.